app/serializers/api/variant_serializer.rb

```diff
@@ -23,4 +22,8 @@ def price_with_fees
   def product_name
     object.product.name
   end
+
+  def tag_list
+    object.tag_list if object.is_a? VariantOverride
+  end
```

Comment:
Just a question here. Is `tag_list` implemented in every "type" of variant or just in `VariantOverride`?

If the latter case we can use duck typing here instead of asking for the "type".